client/src/ui/Landingnavbar.tsx

```diff
@@ -0,0 +1,75 @@
+"use client";
+import Image from "next/image"
+import Link from "next/link"
+import { Button, Typography } from "@mui/material";
+import "./styling/landingnavbar.css"
+import { customTheme } from "@/theme/theme";
+import {ThemeProvider} from "@mui/material";
+
+export default function Landingnavbar(){
+
+    return(
+    <nav className="navbar">
```

Comment:
Please keep the styles in the components itself instead of using classNames. As per the best practices doc, we don't want to abstract the styles in the stylesheets so it's easier for the developers to make changes by not letting them hunt the stylesheets